js/contact-form.js

```diff
@@ -88,9 +93,17 @@ form.addEventListener('submit', (e) => {
   const isMessageValid = checkMessage();
   const isFormValid = isUsernameValid && isEmailValid && isMessageValid;
 
+  const userInfo = {
+    fullName: fullName.value,
+    email: email.value,
+    message: message.value,
+  };
+
   if (isFormValid) {
-    form.submit();
+    window.localStorage.setItem('userInfo', JSON.stringify(userInfo));
+    return form.submit();
   }
+  return false;
 });
```

Comment:
- [ ] Kindly be noticed that you should save the data after filling out the form, and not submit it.
- [ ] you can do this by add events to each input and textarea and update their value after change it.